processing/data_collection/gazette/spiders/pr_sao_jose_pinhais.py

```diff
@@ -0,0 +1,45 @@
+import re
+import dateparser
+import w3lib.url
+
+from datetime import datetime
+from scrapy import Request, Spider
+from gazette.items import Gazette
+
```

Comment:
```
from gazette.spiders.base import BaseGazetteSpider

class PrSaoJosePinhaisSpider(BaseGazetteSpider):
```
Inherit your spider from the base class and the pipeline will filter the gazettes from the period of interest (#46), today it is right, but in the future it can change and will be easier to maintain =).